pallets/pools/src/tranche.rs

```diff
@@ -1344,57 +1346,6 @@ where
 		})
 	}
 
-	pub fn acc_supply_with_fulfillment(
-		&self,
-		fulfillments: &[TrancheSolution],
-	) -> Result<Balance, DispatchError> {
-		self.supplies_with_fulfillment(fulfillments)?
-			.iter()
-			.fold(Some(Balance::zero()), |acc, add| {
-				acc.and_then(|sum| sum.checked_add(add))
-			})
-			.ok_or(ArithmeticError::Overflow.into())
-	}
-
-	pub fn supplies(&self) -> Vec<Balance> {
-		self.tranches.iter().map(|tranche| tranche.supply).collect()
-	}
-
-	pub fn acc_supply(&self) -> Result<Balance, DispatchError> {
-		self.tranches
-			.iter()
-			.fold(Some(Balance::zero()), |sum, tranche| {
-				sum.and_then(|acc| acc.checked_add(&tranche.supply))
-			})
-			.ok_or(ArithmeticError::Overflow.into())
-	}
-
-	pub fn investments(&self) -> Vec<Balance> {
-		self.tranches.iter().map(|tranche| tranche.invest).collect()
-	}
-
-	pub fn acc_investments(&self) -> Result<Balance, DispatchError> {
-		self.tranches
-			.iter()
-			.fold(Some(Balance::zero()), |sum, tranche| {
-				sum.and_then(|acc| acc.checked_add(&tranche.invest))
-			})
-			.ok_or(ArithmeticError::Overflow.into())
-	}
-
-	pub fn redemptions(&self) -> Vec<Balance> {
-		self.tranches.iter().map(|tranche| tranche.redeem).collect()
-	}
-
-	pub fn acc_redemptions(&self) -> Result<Balance, DispatchError> {
-		self.tranches
-			.iter()
-			.fold(Some(Balance::zero()), |sum, tranche| {
-				sum.and_then(|acc| acc.checked_add(&tranche.redeem))
-			})
-			.ok_or(ArithmeticError::Overflow.into())
-	}
-
 	pub fn calculate_weights(&self) -> Vec<(Weight, Weight)> {
```

Comment:
This is unrelated to this PR, but I noticed `calculate_weights` is duplicated in this file. Can/should we clean that up? If so, should we create a ticket for that?

Reply:
Removed.